desktop-widgets/btdeviceselectiondialog.h

```diff
@@ -55,7 +56,7 @@ class BtDeviceSelectionDialog : public QDialog {
 	Q_OBJECT
 
 public:
-	explicit BtDeviceSelectionDialog(QWidget *parent = 0);
+	explicit BtDeviceSelectionDialog(const QString &address, dc_descriptor_t *dc, QWidget *parent = 0);
```

Comment:
Are you sure about that? I looked at two random QObjects and in both cases the parent parameter is last:
http://doc.qt.io/qt-5/qbluetoothlocaldevice.html
http://doc.qt.io/qt-5/qfilesystemwatcher.html
Which kind of makes sense, because only then does the default (nullptr) work.

Reply:
Hmm.. It wasn't as consistent as I thought it looked, and its the sane way to get the default value to work.

Reply:
Official Qt policy puts QObject parent always last. Every other case is wrong but can probably not be fixed due to BC concerns in Qt.